src/components/SpacemeshApiClientMock/index.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from "react";
+import { ISpacemeshApi } from '../../model';
+import { Transaction, TransactionStatus } from 'src/types';
+
+interface IProps extends ISpacemeshApi{
+    apiReady: boolean;
+}
+
+interface IState {
+    connectedToApi: boolean;
+}
+
+const SpacemeshApiClientMock = <P extends IProps>(Component: React.ComponentType<P>) => {
+    return class SpacemeshApiClientMock extends React.Component<P & IProps, IState> implements ISpacemeshApi {
+        constructor(props: P & IProps) {
+            super(props);
+            this.state = {
+                connectedToApi: false,
+            }
+        }
+
+        /**
+         * Broadcast a transaction to the Spacemesh newwork.
+         */
+        public postTransaction = (tx: Transaction): Promise<boolean> => {
```

Comment:
I think that with the default ts-lint rules - all async function name should end with Async and that this is a good pattern.